packaging/release/changelogs/changelog.py

```diff
@@ -175,6 +175,8 @@ def load_plugins(version, force_reload):
         LOGGER.info('refreshing plugin cache')
 
         plugins_data['version'] = version
+
+        plugins_data['plugins'] = {}
```

Comment:
https://github.com/ansible/ansible/pull/53792 will fix this and enforce the correct types for sections.